components/http_client.rst

```diff
@@ -265,6 +265,9 @@ following methods::
     // you can get individual info too
     $startTime = $response->getInfo('start_time');
 
+.. tip::
+    Call ``$response->getInfo('debug')`` to get detailed logs about the HTTP transaction.
```

Comment:
There should be an empty line between the start of the directive (`.. tip::`) and its content.